src/connectors/qq-music.ts

```diff
@@ -5,7 +5,7 @@ Connector.playerSelector = '.player__ft';
 Connector.getTrackArt = () => {
 	const src = Util.getAttrFromSelectors('.song_info__pic', 'src');
 	const trackArt = src?.split('?')[0];
-	return trackArt;
+	return 'https:' + trackArt;
```

Comment:
Do we really need to remove everything after the `?` for the image to load?  If not would replacing the whole `Connector.getTrackArt` functiion with `Connector.trackArtSelector = '.song_info__pic';` work?

By doing this we call some internal logic that extracts the image and normalizes it with the protocol.